rust/json_object_array.rs

```diff
@@ -0,0 +1,26 @@
+//cargo-deps: json="0.12.4"
+
+// Script outputs arrays of objects as JSON
+use std::env;
+use json::JsonValue;
+use json::object;
+
+extern crate json;
+fn main() {
+    let args: Vec<String> = env::args().skip(1).collect();
+
+    let mut json_array = JsonValue::new_array();
```

Comment:
Instead of `let mut json_array = ...` and using a for-loop to push items onto it, we should use `let json_array: JsonValue = args.iter.map(...).collect::<Vec<_>>().into()`